orchagent/bufferorch.cpp

```diff
@@ -39,6 +62,9 @@ void BufferOrch::initTableHandlers()
     m_bufferHandlerMap.insert(buffer_handler_pair(APP_BUFFER_PG_TABLE_NAME, &BufferOrch::processPriorityGroup));
     m_bufferHandlerMap.insert(buffer_handler_pair(APP_BUFFER_PORT_INGRESS_PROFILE_LIST_NAME, &BufferOrch::processIngressBufferProfileList));
     m_bufferHandlerMap.insert(buffer_handler_pair(APP_BUFFER_PORT_EGRESS_PROFILE_LIST_NAME, &BufferOrch::processEgressBufferProfileList));
+    m_bufferHandlerMap.insert(buffer_handler_pair(APP_BUFFER_PORT_CONFIG_TO_PG_PROFILE_TABLE, &BufferOrch::processPortConfigToPgProfile));
+    m_bufferHandlerMap.insert(buffer_handler_pair(APP_BUFFER_PORT_CABLE_LENGTH_TABLE, &BufferOrch::processPortCableLenth));
+
```

Comment:
Is this empty line at the end of function BufferOrch::initTableHandlers()? If yes, please remove it.